routes/hot_feed.go

```diff
@@ -775,6 +1038,29 @@ func (fes *APIServer) AdminUpdateHotFeedAlgorithm(ww http.ResponseWriter, req *h
 		}
 	}
 
+	if requestData.TimeDecayBlocksTag > 0 {
+		err := fes.GlobalState.Put(
+			_GlobalStatePrefixForHotFeedTagTimeDecayBlocks,
+			lib.EncodeUint64(uint64(requestData.TimeDecayBlocksTag)),
+		)
+		if err != nil {
+			_AddInternalServerError(ww, fmt.Sprintf("AdminUpdateHotFeedAlgorithm: Error putting TimeDecayBlocks: %v", err))
+			return
+		}
+	}
+
+	if len(requestData.TxnTypeMultiplierMap) > 0 {
+		txnTypeMultiplierMapBuffer := bytes.NewBuffer([]byte{})
+		if err := gob.NewEncoder(txnTypeMultiplierMapBuffer).Encode(requestData.TxnTypeMultiplierMap); err != nil {
+			_AddBadRequestError(ww, fmt.Sprintf("AdminUpdateHotFeedAlgorithm: Problem encoding transaction multiplier map: %v", err))
+			return
+		}
+		if err := fes.GlobalState.Put(_GlobalStatePrefixHotFeedTxnTypeMultiplierBasisPoints, txnTypeMultiplierMapBuffer.Bytes()); err != nil {
```

Comment:
does it make sense for us to just store this in the frontend cache we use so that the value is updated immediately + we don't have to hit global state everytime the goroutine runs?

Reply:
applies to all these constants